cmd/openshift-install/version.go

```diff
@@ -24,5 +25,8 @@ func runVersionCmd(cmd *cobra.Command, args []string) error {
 	if version.Commit != "" {
 		fmt.Printf("built from commit %s\n", version.Commit)
 	}
+	if image, ok := bootstrap.LockedReleaseImage(); ok {
+		fmt.Printf("release image %s\n", image)
```

Comment:
Why not rename `defaultReleaseImage` to a public `DefaultReleaseImage` (or `LockedReleaseImage`, or just `ReleaseImage`) and use that here?  I don't see a need for a wrapping `LockedReleaseImage` whose only purpose is to collapse a rich `error` into a binary `bool`.

Reply:
Updated